build_darwin.sh

```diff
@@ -1,12 +1,14 @@
 #!/bin/sh
 set -eu
 
+jobs=$(sysctl -n hw.logicalcpu_max)
+
```

Comment:
Can this be taken as an argument instead, and only fall back to hw.logicalcpu_max as a default?  (Generally I try to avoid hard-coding logic to unconditionally use all CPUs: if you're trying to do multiple things at a time, you might want to limit them to smaller subsets of the CPUs rather than have them competing with each other -- and the web browser you're trying to use to read Twitter while you run your builds -- over all the CPUs.)

Reply:
[not blocking; would be nice]